### PR TITLE
prevent react key message

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
inner non blocking bug that create log message when the page is HTML and has head tags

### CODE REVIEW

File: frontend/taipy-gui/src/components/pages/TaipyRendered.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ const TaipyRendered = (props: TaipyRenderedProps) => {
     return (
         <ErrorBoundary FallbackComponent={ErrorFallback}>
             {head.length ? (
-                <Helmet>{head.map((v) => React.createElement(v.tag, v.props, v.content))}</Helmet>
+                <Helmet>{head.map((v,i) => React.createElement(v.tag, {key: `head${i}`, ...v.props}, v.content))}</Helmet>
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/components/pages/TaipyRendered.tsx]:
```suggestion
                <Helmet>{head.map((v, i) => React.createElement(v.tag, {key: `head${i}`, ...v.props}, v.content))}</Helmet>
```

**Review by dinhlongviolin1 — DISMISSED** [review]
LGTM

**Review by FabienLelaquais — DISMISSED** [review]
👌

**Review by dinhlongviolin1 — APPROVED** [review]
tx lint for making the PR a bit longer

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**